terraform-modules/aws/istio-networking/main-gateway/variables.tf

```diff
@@ -28,7 +28,13 @@ variable "cert_dns_name" {
   type = string
   description = "The dns name for the certificate"
 }
-  
+
+variable "enable_certificate" {
+  type = bool
+  description = "If set to true, it will create the certificate resource on-demand"
+  default = true
+}
```

Comment:
The default value is true to maintain compatibility with customers currently using this module.

Reply:
Documentation of default values: [default](https://learn.hashicorp.com/tutorials/terraform/aws-variables#:~:text=default%20%20%20%20%20%3D%20%22ExampleAppServerInstance%22  )